sdk/lib/crt/math/i386/ftol2_asm.s

```diff
@@ -25,4 +24,59 @@ __ftol2:
 __ftol2_sse:
     jmp __ftol
 
+__real@43e0000000000000:
+    .quad HEX(43e0000000000000)
+
+__ftoul2_legacy:
+
+    /* Compare the fp number, passed in st(0), against (LLONG_MAX + 1)
+       aka 9223372036854775808.0 (which is 0x43e0000000000000 in double format).
```

Comment:
```suggestion
       aka. 9223372036854775808.0 (which is 0x43e0000000000000 in double format).
```

Reply:
Grammar fight!

You can't say "aka." here, it's "aka" or "a.k.a." (with periods is technically the correct one (the best kind of correct) but without periods is more popular according to my research).